plugins/flytekit-inference/flytekitplugins/inference/ollama/serve.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         cpu: int = 1,
         gpu: int = 1,
         mem: str = "15Gi",
+        inputs_mem: str = "500Mi",
```

Comment:
I prefer be more explicit: `download_inputs_mem`.

```suggestion
        download_inputs_mem: str = "500Mi",
```